### PR TITLE
Merges the Ripley MK-II and Firefighter

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

Merges the Ripley MK-II and Firefighter into the same mech, giving the former the same functionality and appearance as the latter.

# Why is this good for the game?

There's not really any good reason to keep them separate, they're almost identical.

# Testing

image

image

# Wiki Documentation

On the guide to robotics, remove the Ripley MK-II section and change the Firefighter section to the same steps as the MK-II was.

# Changelog

:cl:  
tweak: Ripley MK-II and Firefighter have been merged into the same mech
/:cl:

### CODE REVIEW

File: code/__DEFINES/exosuit_fab.dm [hunk 1]
```diff
@@ -15,8 +15,6 @@
 #define EXOSUIT_MODULE_RIPLEY		(1<<0)
 /// Module is compatible with Odyseeus Exosuit models
 #define EXOSUIT_MODULE_ODYSSEUS		(1<<1)
-/// Module is compatible with Firefighter Exosuit models
-#define EXOSUIT_MODULE_FIREFIGHTER	(1<<2)
 /// Module is compatible with Gygax Exosuit models
 #define EXOSUIT_MODULE_GYGAX		(1<<3)
```
Comment (Moltijoe) [inline code/__DEFINES/exosuit_fab.dm]: don't forget to update the other flags

Comment (iamthesilly) [reply code/__DEFINES/exosuit_fab.dm]: i forgor

File: code/__DEFINES/exosuit_fab.dm [hunk 2]
```diff
@@ -15,21 +15,19 @@
...
 /// Module is compatible with Phazon Exosuit models
-#define EXOSUIT_MODULE_PHAZON		(1<<6)
+#define EXOSUIT_MODULE_PHAZON		(1<<5)
 /// Module is compatible with Sidewinder Exosuit models
-#define EXOSUIT_MODULE_SIDEWINDER	(1<<7)
+#define EXOSUIT_MODULE_SIDEWINDER	(1<<6)
 
 /// Module is compatible with "Working" Exosuit models - Ripley and Firefighter
-#define EXOSUIT_MODULE_WORKING		EXOSUIT_MODULE_RIPLEY | EXOSUIT_MODULE_FIREFIGHTER
+#define EXOSUIT_MODULE_WORKING		EXOSUIT_MODULE_RIPLEY
```
Comment (JohnFulpWillard) [inline code/__DEFINES/exosuit_fab.dm:30]: this flag seems worthless now

**Comment by githubuser4141** [comment]
They're not almost identical.. the MK II is faster, has less armour/health and more gear slots then the Firefighter. It's good when you just want a Ripley with space proofing. The firefighter is distinctive because it's really only specialised for fires/mining. It's *worse* at general utility because of the aforementioned speed and slots. At the same time Clarke isn't a full Ripley replacement as its unable to use hydraulic clamps to open fire locks/airlocks or pick things up.

**Comment by githubuser4141** [comment]
Well nevermind, for some reason in history the Firefighter was "balanced" to have the same speed as the MK II despite having superior armour, health and environmental protection... and I see in this PR it gives it more gear too..?

**Review by Aquizit — APPROVED** [review]

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**